data/json/items/comestibles.json

```diff
@@ -10192,6 +10192,14 @@
     "spoils_in": 420,
     "use_action": "WEED_BROWNIE",
     "flags": [ "EATEN_HOT" ]
+  },
+   {
+    "type": "COMESTIBLE",
+    "id": "hflesh_pemmican",
+    "name": "prepper pemmican",
+    "copy-from": "pemmican",
```

Comment:
Good use of "copy-from", additionally, you may want to stick this entry underneath the pemmican one (most cannibal recipes generally follow this).